routing/routing_tests/applying_traffic_test.cpp

```diff
@@ -0,0 +1,143 @@
+  loader->AddRoad(7 /* featureId */, true /* oneWay */, 1.0 /* speed */,
+                  RoadGeometry::Points({{2.0, 0.0}, {3.0, 0.0}}));
+  loader->AddRoad(8 /* featureId */, true /* oneWay */, 1.0 /* speed */,
+                  RoadGeometry::Points({{3.0, 0.0}, {3.0, 1.0}}));
+
+  vector<Joint> const joints = {
+      MakeJoint({{0 /* feature id */, 0 /* point id */}}), /* joint at point (0, 0) */
+      MakeJoint({{1, 0}, {7, 0}}),                         /* joint at point (2, 0) */
+      MakeJoint({{0, 1}, {1, 1}, {2, 0}, {3, 0}}),         /* joint at point (1, 1) */
+      MakeJoint({{2, 1}}),                                 /* joint at point (0, 2) */
+      MakeJoint({{3, 1}, {4, 1}, {5, 0}, {6, 0}}),         /* joint at point (2, 2) */
+      MakeJoint({{4, 0}, {8, 1}}),                         /* joint at point (3, 1) */
+      MakeJoint({{5, 1}}),                                 /* joint at point (1, 3) */
+      MakeJoint({{6, 1}}),                                 /* joint at point (3, 3) */
+      MakeJoint({{7, 1}, {8, 0}}),                         /* joint at point (3, 0) */
+  };
+
+  unique_ptr<IndexGraph> graph = make_unique<IndexGraph>(move(loader), estimator);
+  graph->Import(joints);
+  return graph;
+}
+
+// Route through XX graph without any traffic info.
+UNIT_TEST(XXGraph_EmptyTrafficColoring)
+{
+  classificator::Load();
+  shared_ptr<EdgeEstimator> estimator =
+      EdgeEstimator::CreateForCar(*make_shared<CarModelFactory>()->GetVehicleModel());
+
+  unique_ptr<IndexGraph> graph = BuildXXGraph(estimator);
+  IndexGraphStarter starter(*graph, RoadPoint(1, 0) /* start */, RoadPoint(6, 1) /* finish */);
+  vector<m2::PointD> const expectedGeom = {{2 /* x */, 0 /* y */}, {1, 1}, {2, 2}, {3, 3}};
+  TestRouteGeometry(starter, AStarAlgorithm<IndexGraphStarter>::Result::OK, expectedGeom);
+}
+
+// Route through XX graph with SpeedGroup::G0 on F3.
+UNIT_TEST(XXGraph_G0onF3)
+{
+  classificator::Load();
+  shared_ptr<EdgeEstimator> estimator =
+      EdgeEstimator::CreateForCar(*make_shared<CarModelFactory>()->GetVehicleModel());
+  TrafficInfo::Coloring coloring = {
+      {{3 /* feature id */, 0 /* segment id */, TrafficInfo::RoadSegmentId::kForwardDirection},
+       SpeedGroup::G0}};
+  shared_ptr<TrafficInfo> trafficInfo = make_shared<TrafficInfo>();
+  trafficInfo->SetColoringForTesting(coloring);
+  estimator->SetTrafficInfo(trafficInfo);
+
+  unique_ptr<IndexGraph> graph = BuildXXGraph(estimator);
+  IndexGraphStarter starter(*graph, RoadPoint(1, 0) /* start */, RoadPoint(6, 1) /* finish */);
+  vector<m2::PointD> const expectedGeom = {{2 /* x */, 0 /* y */}, {3, 0}, {3, 1}, {2, 2}, {3, 3}};
+  TestRouteGeometry(starter, AStarAlgorithm<IndexGraphStarter>::Result::OK, expectedGeom);
+}
+
+// Route through XX graph SpeedGroup::G1 on F3 and F6, SpeedGroup::G4 on F8 and F4.
+UNIT_TEST(XXGraph_G0onF3andF6andG4onF8andF4)
+{
+  shared_ptr<EdgeEstimator> estimator =
```

Comment:
In all three tests you create EdgeEstimator for CarModelFactory. Also, looks like you need to load classificator in all three tests. Could you please switch from UNIT_TESTS to UNIT_CLASS_TESTS?